src/pyggdrasil/distances/_scphylo_wrapper.py

```diff
@@ -8,7 +10,10 @@
 
 
 class AncestorDescendantSimilarity(interface.TreeSimilarity):
-    """Ancestor-descendant accuracy."""
+    """Ancestor-descendant accuracy.
```

Comment:
At this stage I'm actually quite convinced that we should just remove this wrapper from our codebase. Would you mind submitting an issue to [scphylo](https://github.com/faridrashidi/scphylo-tools), with the discrepancy you observed? They already have an issue regarding R–F distance, so perhaps we will at some pint move away from using this entirely... (But if MP3 distance is working fine, then we can leave it for now.)

Reply:
Opened an issue here 
- https://github.com/faridrashidi/scphylo-tools/issues/8
and documented as well as I could.

I'll merge this feature as it is now.